base_layer/wallet/src/transaction_service/protocols/transaction_coinbase_monitoring_protocol.rs

```diff
@@ -119,6 +118,7 @@ where TBackend: TransactionBackend + 'static
         //      a) If the chain tip moves beyond this block height AND output is not in the Utxo set cancel this Tx
         //      b) If the output is in the Utxo set the protocol can end with success
         //      c) Timeout is reached > Start again
+        let mut shutdown = self.resources.shutdown_signal.clone();
```

Comment:
The comment above probably needs to be updated to be kept up to date with these changes, at the very least regarding the fact that the protocol continues running until the coinbase is confirmed.